src/xdocs/google_style.xml

```diff
@@ -1282,7 +1282,7 @@
                             <td><a name="6.2"/><a href="#6.2"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s6.2-caught-exceptions">6.2 Caught exceptions: not ignored</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s6.2-caught-exceptions">6.2 Caught exceptions: not ignored</a>
```

Comment:
not sure why we have green image there.
please make it "Existing Check covers some part of requirements from Google." as we can not cover Exception case.
Looks like ITs input file is too smiple - please append more cases for "expected"(exact match and begin with) to https://github.com/checkstyle/checkstyle/blob/master/src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule62donotignoreexceptions/InputEmptyBlockCatch.java